velox/docs/functions/spark/math.rst

```diff
@@ -96,6 +96,14 @@ Mathematical Functions
 
     Returns Euler's number raised to the power of ``x``.
 
+.. spark:function:: expm1(x) -> double
+
+    Returns Euler's number raised to the power of ``x`` minus 1 and this function is more accurate than exp(num) - 1 when the num is close to zero.
```

Comment:
Does it mean exp(x) - 1 or exp(x - 1)? Better to provide mathematic representation to avoid misunderstanding.

Reply:
exp(x) - 1 and add more comments here.